chef/data_bags/crowbar/migrate/nova_dashboard/026_use_keystone_session_timeout.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  keystone_instance = a['keystone_instance']
+  proposal_obj = ProposalObject.find_proposal("keystone", keystone_instance)
+  keystone_timeout = (proposal_obj["attributes"]["keystone"]["token_expiration"] || 3600)/60
```

Comment:
Because it's the one in the keystone data bag that will be used for the keystone migration.